docs/conf.py

```diff
@@ -38,6 +38,10 @@
 ## lets us suppress the copy button on select code blocks.
 copybutton_selector = "div:not(.no-copybutton) > div.highlight > pre"
 
+# -- Options for autosectionlabel -------------------------------------------------
+
+autosectionlabel_prefix_document=True
```

Comment:
ooo, what does this do?

Reply:
This makes it so we can have a section in the PyPI documentation called "Set-up" and one in the conda page with the same name, but still be able to create the references. Otherwise, the section names would need to be unique throughout the whole site for the cross-referencing to work.

https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#confval-autosectionlabel_prefix_document